.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+
+node_js:
+  - 6
+
+before_script:
+  - npm run lint
```

Comment:
Conversely, we can put the lint step in the package.json as a `"pretest": "npm run lint"` so running `npm test` locally will also trigger the linting task.